proof/crefine/RISCV64/Ipc_C.thy

```diff
@@ -4142,7 +4475,17 @@ lemma handleFaultReply_ccorres [corres]:
             apply (vcg exspec=copyMRsFaultReply_modifies)
            apply (wpsimp wp: threadGet_wp)+
          (* Timeout *)
-         subgoal sorry (* FIXME RT: Timeout case for handleFaultReply_ccorres *)
+         apply (rename_tac number)
```

Comment:
number?

Reply:
I think this is copy paste from the previous case. I guess it should be `badge`, but I can check this.

Reply:
If it's copy/paste then fine to leave as is

Reply:
Turns out the green variable wasn't even used, so I've removed this `rename_tac`